Dockerfile

```
@@ -58,7 +58,7 @@ SHELL ["mamba", "run", "-n", "streamlit-env", "/bin/bash", "-c"]
 
 # Install up-to-date cmake via mamba and packages for pyOpenMS build.
 RUN mamba install cmake
-RUN pip install --upgrade pip && python -m pip install -U setuptools nose 'cython<3.1' autowrap pandas numpy pytest
+RUN pip install --upgrade pip && python -m pip install -U setuptools nose cython autowrap pandas numpy pytest
```

Hadolint (2.12.0)

[error] Ensure the shebang uses an absolute path to the interpreter.

(SC2239)